clickhouse-data/src/main/java9/com/clickhouse/data/ByteUtils.java

```diff
@@ -7,7 +7,7 @@
 
 public final class ByteUtils {
     public boolean equals(byte[] a, int aFromIndex, int aToIndex, byte[] b, int bFromIndex, int bToIndex) {
-        return Arrays.equals(a, aFromIndex, aToIndex, a, bFromIndex, bToIndex);
+        return Arrays.equals(a, aFromIndex, aToIndex, b, bFromIndex, bToIndex);
```

Comment:
@chernser I guess we don't have any tests for the methog

Reply:
@mshustov I will take care of it this time.